.gitignore

```diff
@@ -10,3 +10,6 @@ working_dir
 __pycache__/
 *.py[cod]
 dist/
+
+.idea/
+.python-version
```

Comment:
playing around with python versions and pycharm a little exploring the code.  felt appropriate to add these ignores to avoid me from pushing accidental clutter